src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,5 @@
-export const Tabs = () => {};
+export const Tabs = ({ content }) => (
+  <div className="block" data-cy="TabContent">
+    {content}
+  </div>
+);
```

Comment:
<img width="907" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/70220667/6a96b876-fef3-431d-bedd-6f4bc57ec694">
you should implement this component as written here